projects/components/src/datagrid/datagrid.component.ts

```diff
@@ -554,21 +559,9 @@ export class DatagridComponent<R> implements OnInit, AfterViewInit {
     private viewInitted = false;
 
     /**
-     * Gives the correct string to display for the pagination.
-     *
-     * @param firstItem the index of the first item displayed.
-     * @param lastItem the index of the last item displayed.
-     * @param totalItems the total number of items that could be displayed.
+     * Used for translating pagination information displayed in the grid
```

Comment:
Can you add some information about what data is passed along with the call to `translate`? I.E. firstItem, lastItem, totalItems

Reply:
The `getPaginationTranslation` method that actually calls the `translate`, uses `ClrDatagridPagination` given by the HTML and extracts those 3 params from it to pass them to the `translate` method.

As the user did not call the `` method directly with those parameters, I don't think they require to be documented.